web/styles.css

```diff
@@ -131,18 +140,28 @@ body {
 }
 
 .github-link {
+    position: absolute;
```

Comment:
moved in the absolute positioning, top, and left from raw styles applied to the link in the html